From 83645fe617c13a79731c511bc0a1742b8a8ddad7 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Thu, 19 Sep 2013 17:11:52 +0300
Subject: [PORT FROM MAIN] ril: wrong values of intel ril apis

BZ: 139736

The values of these apis will be continous in java framework
side, and so should be continous in ril.h too.

Change-Id: If430285bbfc217163ba2027cc300f2d55b4b289b
Orig-Change-Id: I475635f3def66f2fc0745e96036ceeef73668ede
Category: aosp improvement
Domain: Telephony-Telephony-FWK
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 include/telephony/ril.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/telephony/ril.h b/include/telephony/ril.h
index 4e5eaf8..e196b2c 100644
--- a/include/telephony/ril.h
+++ b/include/telephony/ril.h
@@ -3634,7 +3634,7 @@ typedef struct {
  *  GENERIC_FAILURE
  *
  */
-#define RIL_REQUEST_HANGUP_VT 118
+#define RIL_REQUEST_HANGUP_VT 120
 
 /**
  * RIL_REQUEST_DIAL_VT
@@ -3651,7 +3651,7 @@ typedef struct {
  *  GENERIC_FAILURE
  *
  */
-#define RIL_REQUEST_DIAL_VT 119
+#define RIL_REQUEST_DIAL_VT 121
 
 #endif  //(M2_VT_FEATURE_ENABLED)
 
@@ -3673,7 +3673,7 @@ typedef struct {
  * RADIO_NOT_AVAILABLE (radio resetting)
  * GENERIC_FAILURE, if an error occurred
  */
-#define RIL_REQUEST_GET_SIM_SMS_STORAGE 120
+#define RIL_REQUEST_GET_SIM_SMS_STORAGE 122
 
 #endif // M2_GET_SIM_SMS_STORAGE_ENABLED
 
-- 
1.7.4.1

